pmd-cataloger/src/main/java/sfdc/sfdx/scanner/messaging/SfdxMessager.java

```diff
@@ -34,33 +39,50 @@ public void uxWarn(EventKey key, List<String> args) {
   }
 
   public void uxWarn(EventKey key, List<String> args, boolean verbose) {
+    // TODO: delete in the next iteration
     System.err.println(formatMessage(key, args, MessageType.WARNING, MessageHandler.UX, verbose));
+    final Message message = new Message(key, args, "", MessageType.WARNING, MessageHandler.UX, verbose);
```

Comment:
Going to stop making suggestions now...you get the idea ;). Use the spread syntax instead of arrays...more usable for the caller

Reply:
Good point! Changes coming up.